src/components/molecules/RadioGroupField/RadioGroupField.test.tsx

```diff
@@ -0,0 +1,71 @@
+import React, { useState } from 'react';
+import { act, fireEvent, render } from '@testing-library/react';
+import { axe } from 'jest-axe';
+import RadioGroupField from './RadioGroupField';
+
+describe('<RadioGroupField />', () => {
+  it('should fire handle value change if it is not controlled', () => {
+    const onChange = jest.fn();
+    const { getByLabelText } = render(
+      <RadioGroupField
+        label="label"
+        onChange={onChange}
+        items={[
+          {
+            value: 'one',
+            label: 'label one',
+            defaultChecked: true,
+          },
+          {
+            value: 'two',
+            label: 'label two',
+          },
+        ]}
+      />,
+    );
+    const labelOne: any = getByLabelText('label one');
+    const labelTwo: any = getByLabelText('label two');
+    expect(labelOne.checked).toEqual(true);
+    expect(labelTwo.checked).toEqual(false);
+
+    fireEvent.click(labelTwo);
+    expect(onChange).toHaveBeenCalledWith('two');
+    expect(labelOne.checked).toEqual(false);
+    expect(labelTwo.checked).toEqual(true);
+  });
+
+  it('should fire onChange with a correct value if is not controlled', () => {
```

Comment:
```
should not handle value change by itself if controlled
```
The test is not testing that `onChange` was called though, should we test that? 🤔

Reply:
Changed the description to be more generic. However, I'm not sure if it makes sense to check if `onChange` gets called here since mocking it would mean that the value doesn't change... seems a bit awkward